.github/workflows/release-prod.yaml

```diff
@@ -17,8 +17,7 @@ jobs:
         id: tag_version
         uses: mathieudutour/github-tag-action@v5.5
         with:
-          # Use GITHUB_TOKEN here to prevent further workflows
-          # generated on 'tag' action
+          dry_run: true
```

Comment:
Just a question, why was this set to dry_run?

Reply:
We want the NITRIC_BOT_TOKEN to push to tag as part of the release, so the tag push action that publishes the binaries get's triggered.